requirements.txt

```diff
@@ -11,3 +11,5 @@ dj-static
 raven
 python-dateutil
 Faker
+mypy==0.700
```

Comment:
Why version requirement is so strict?

Reply:
django-stubs currently supports only this version of mypy

Reply:
https://pypi.org/project/django-stubs/: `Supports Python 3.6/3.7, and Django 2.1.x series.`

django-stubs doesn't support our version of Django!

Reply:
```
Could be run on earlier versions of Django, but expect some missing imports warnings.
```